### PR TITLE
Fix fatal error if setup is called under 'if __name__ == "__main__":'

### PR DESCRIPTION
**Pull request by lhupfeldt**

### CODE REVIEW

**Comment by lhupfeldt** [comment]
Travis failed for python 2.6, not sure how it is related to my PR.

**Comment by jirikuncar** [comment]
The travis fail is related to `pydocstyle`. IMHO we can remove the Python 2.6 from tests.

File: tests/data/setup_if_main.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Requirements-Builder
+# Copyright (C) 2015, 2016, 2017 CERN.
```
Comment (jirikuncar) [inline tests/data/setup_if_main.py]: since it's new file you should only add 2017

Comment (lhupfeldt) [inline tests/data/setup_if_main.py]: Done

**Comment by lhupfeldt** [comment]
Hi Jiri, Who else need to review these pull requests?

**Comment by jirikuncar** [comment]
@lhupfeldt we need to wait for @lnielsen or @egabancho. I hope they get back to you (us) soon.

**Comment by lnielsen** [comment]
Thanks!

**Comment by lhupfeldt** [comment]
Hi Lars, Thank you for merging this. Are you also going to create a new release? I very much want to start using this great package, but I need it to be released so that I can pip install it on Travis.
p.s. I noticed that the latest release which shows up here is 0.2.0, but 0.2.5 is the latest release on pypi.

**Comment by lnielsen** [comment]
Once [Travis passes](https://travis-ci.org/inveniosoftware/requirements-builder/builds/253345841) you should have a release on PyPI. Cheers, Lars.

**Pull request merged**